packages/ra-ui-materialui/src/input/CheckboxGroupInput.tsx

```diff
@@ -235,16 +235,20 @@ export const CheckboxGroupInput: FunctionComponent<CheckboxGroupInputProps> = pr
                     />
                 ))}
             </FormGroup>
-            <FormHelperText
-                error={fetchError || ((isTouched || isSubmitted) && !!error)}
-                className={CheckboxGroupInputClasses.helperText}
-            >
-                <InputHelperText
-                    touched={isTouched || isSubmitted || fetchError}
-                    error={error?.message || fetchError?.message}
-                    helperText={helperText}
-                />
-            </FormHelperText>
+            {helperText !== false ? (
```

Comment:
I disagree with this implementation. To me we should display the `<FormHelperText>` when there is a validation error, even when `helperText === false`